coreclient/src/conversations/messages/mod.rs

```diff
@@ -265,14 +289,19 @@ impl Message {
 // introduced and the storage logic changed accordingly.
 #[derive(PartialEq, Debug, Clone, Serialize, Deserialize)]
 pub struct ContentMessage {
+    pub(super) mimi_id: Option<MimiId>,
     pub(super) sender: UserId,
     pub(super) sent: bool,
     pub(super) content: MimiContent,
 }
 
 impl ContentMessage {
-    pub fn new(sender: UserId, sent: bool, content: MimiContent) -> Self {
+    pub fn new(sender: UserId, sent: bool, content: MimiContent, group_id: &GroupId) -> Self {
+        let mimi_id = MimiId::calculate(group_id, &sender, &content)
+            .inspect_err(|error| error!(%error, "Failed to calculate Mimi ID"))
```

Comment:
Shouldn't this fail if we can't calculate the MIMI ID? Or are we ignoring the error here because we don't think it can fail? If so, a brief code comment would be helpful.

Reply:
Indeed, it should never fail, except when the content can't be serialized, which is a bug in our software. However, since the content is input from the user, it might happen that the user finds a way to trigger this bug. Therefore, we don't want to panic here. But making this function fallible, would create to much error propagation for nothing.